### DISCUSSION TITLE
How to set color of disabled switch.

### DISCUSSION THREAD

**Discussion by Kowersky** [zauberzeug/nicegui#3569] [Q&A]
### Question

Hi. I need to set custom color when the switch is disabled like this.
![image](https://github.com/user-attachments/assets/786e8a74-2c85-4ffc-817b-7db6597d342a)

And i need to change this color

**Comment by falkoschindler** [comment]
Hi @Kowersky,

I assume you mean the "falsy" state, not the situation when the UI element is disabled.
Since there doesn't seem to be a prop for this case, you might need some custom CSS:
```py
ui.add_css('''
    .custom-toggle .q-toggle__inner--falsy .q-toggle__track {
        background-color: red;
    }
    .custom-toggle .q-toggle__inner--falsy .q-toggle__thumb::after {
        background-color: LightBlue;
    }
''')

ui.switch('Hello, world!').classes('custom-toggle')
```
![Screenshot 2024-08-22 at 14 44 11](https://github.com/user-attachments/assets/25ec4985-6e85-4d84-b65f-7a8a4fbd7bfc)
